### DISCUSSION TITLE
Dash to Panel Extension

### DISCUSSION THREAD

**Discussion by trevormac789** [mjakeman/extension-manager#557] [Q&A]
Hi, we're just about ready to migrate to Rocky Linux 9 and we had Dash to Panel in Centos 7. Is this supported or going to be supported at some point? Or maybe I'm overlooking something I can do myself. Someone on the testing team was requesting it. 

Thanks!

**Comment by oscfdezdz** [comment]
Hello, that depends on the extension itself rather than Extension Manager. Checking quickly, on [Extensions website](https://extensions.gnome.org/extension/1160/dash-to-panel/), Dash to Panel has support for almost all versions of GNOME Shell (including GNOME Shell 40 which is the one Rocky Linux 9 seems to have). Although for that version of GNOME Shell, the update is handled by the official Extensions app. See https://github.com/mjakeman/extension-manager#extensions-are-not-being-updated.

**Reply by trevormac789** [reply]
Alright. Thank you. I'll have a look.
